easybuild/easyconfigs/m/maze/maze-20170124-foss-2020b.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonBundle'
+
+name = 'maze'
+version = '20170124'
+
+homepage = 'https://github.com/dellytools/maze'
+description = """In a similar manner to dot plots, maze highlights local
+sequence similarity between two DNA sequences. In particular, maximal exact
+substring matches are computed with MUMmer3 and visualised."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('MUMmer', '4.0.0beta2'),
+    ('LAST', '1179'),
+    ('Flask', '1.1.2'),
+    ('Pysam', '0.16.0.1'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    ('Click', '7.0', {
```

Comment:
@zarybnicky `click` is already included with `Python` 3.8.6, so you can just use `PythonPackage` for `maze`?